content/abuse/how-much-abuse/contents.lr

```diff
@@ -0,0 +1,16 @@
+title: Does Tor get much abuse?
+---
+key: 5
+---
+description:
+Not much, in the grand scheme of things.
+The network has been running since October 2003, and it's only generated a handful of complaints.
+Of course, like all privacy-oriented networks on the net, it attracts its share of jerks.
+Tor's exit policies help separate the role of "willing to donate resources to the network" from the role of "willing to deal with exit abuse complaints,"
+so we hope our network is more sustainable than past attempts at anonymity networks.
+
+Since Tor has [many good uses as well](https://2019.www.torproject.org/about/torusers.html.en), we feel that we're doing pretty well at striking a balance currently. 
```

Comment:
Let's link here to our brand new user personas instead of the old website:
https://community.torproject.org/user-research/persona/